From 20f1f5e66f08acdca1962cc4f94e3fd70a2e7156 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@foss.st.com>
Date: Mon, 11 Sep 2023 16:56:40 +0200
Subject: [PATCH] arm64: dts: st: increase deassert of PHY reset in
 stm32mp257f-ev1

The RTL8211F has a PHYRSTB pin to reset the chip.
For a complete PHY reset, this pin must be asserted low for
at least 10ms for the internal regulator.
Wait for at least 80ms (for internal circuits setting time) before
accessing the PHY register.
All registers will return to default values after a hardware reset

Signed-off-by: Christophe Roullier <christophe.roullier@foss.st.com>
Change-Id: I1970611b99a74933343b807e37fd28f4e4e9f72d
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 3b577ccb6a87..63625f153eaf 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -228,7 +228,7 @@ phy2_eth1: ethernet-phy@5 {
 				     "ethernet-phy-ieee802.3-c22";
 			reset-gpios =  <&gpioj 9 GPIO_ACTIVE_LOW>;
 			reset-assert-us = <10000>;
-			reset-deassert-us = <300>;
+			reset-deassert-us = <80000>;
 			realtek,eee-disable;
 			reg = <5>;
 		};
@@ -252,7 +252,7 @@ phy1_eth2: ethernet-phy@1 {
 			compatible = "ethernet-phy-id001c.c916";
 			reset-gpios =  <&gpiog 6 GPIO_ACTIVE_LOW>;
 			reset-assert-us = <10000>;
-			reset-deassert-us = <300>;
+			reset-deassert-us = <80000>;
 			realtek,eee-disable;
 			reg = <1>;
 		};
-- 
2.39.5

